quasar-core/src/main/java/co/paralleluniverse/fibers/Fiber.java

```diff
@@ -105,11 +106,19 @@
         assert printVerifyInstrumentationWarning();
     }
 
+    public static void readSystemProperties() {
+        // ENT-5444 To facilitate testing make reading of system properties public.
```

Comment:
I think we could consciously put a comment here saying we didn't bother making the fields `volatile` because this should only be called from tests, the tests don't need it and we don't want the memory barrier in normal when accessing the properties.